Gifski/util.swift

```diff
@@ -2168,3 +2168,9 @@ extension CGSize {
 		return CGSize(width: width.rounded(rule), height: height.rounded(rule))
 	}
 }
+
+extension BinaryFloatingPoint {
+	func isEqual(to otherValue: Self, roundingDecimalPlaces decimalPlaces: Int) -> Bool {
+		return rounded(toDecimalPlaces: decimalPlaces) == otherValue.rounded(toDecimalPlaces: decimalPlaces)
+	}
+}
```

Comment:
I would just use a polyfill of this: https://github.com/apple/swift-evolution/blob/master/proposals/0259-approximately-equal.md

Reply:
Good idea 👍

Reply:
Good idea 👍